packages/shared/web/src/dashboard-view/hooks/use-fixed-rate-grid.tsx

```diff
@@ -27,10 +29,10 @@ export const useFixedRateGrid = (
   const isBorrow = product === PRODUCTS.BORROW_FIXED;
   const totalArbPoints = useTotalArbPoints();
   const currentSeason = useCurrentSeason();
-  const fCashYieldData = useNetworkTokens(network, 'fCash');
-  const yieldData = isBorrow
-    ? fCashYieldData.filter(({ token }) => token.isFCashDebt)
-    : fCashYieldData.filter(({ token }) => !token.isFCashDebt);
+  const currentNetworkStore = useCurrentNetworkStore();
+  const fCashLend = currentNetworkStore.getFCashLendData();
```

Comment:
Since this is not in a hook, you don't need to call both functions, you can just do:
`const yieldData = isBorrow ? getFCashDebtData() : getFCashLendData()